docs/sdk/src/reference_docs/frame_currency.rs

```diff
@@ -1,8 +1,99 @@
 //! FRAME Currency Abstractions and Traits
```

Comment:
This does not cover NFTs which are also tokens. I don't see this potential for confusion, specially it the person reads the document

Reply:
Maybe it does make sense to also to discuss NFT traits and trait impls in this doc. It would be similarly helpful to disambiguate the different non-fungible token pallets and traits, just like it's helpful to disambiguate the different fungible token pallets and traits. 

If we don't I still don't think it should be called 'currency'. A 'currency' is a specific type of fungible tokens, and this doc so far explores traits and trait impls that can be used for much more than just currencies.